dkron/api.go

```diff
@@ -357,14 +362,55 @@ func (h *HTTPTransport) executionsHandler(c *gin.Context) {
 		return
 	}
 
+	if output_size > 0 {
```

Comment:
The fallback for output_size is -1 according to code above.
This check also falls back to returning everything when output_size is 0. Is that intended?

Reply:
yes, it was intended as I thought it make no sense to allow 0 output_size.
But thinking of it now, maybe it does make sense to support "no output" / `limit=0` ? what do you think?

Reply:
changed it in the latest commit :)